custom.sample.php

```diff
@@ -15,16 +16,206 @@
+      <td><?php print round(($last30m/1000),3); print " GH/s"; ?></td>
+      <td><?php print round(($last1h/1000),3); print " GH/s"; ?></td>
+      <td><?php print round(($last1d/1000),3); print " GH/s"; ?></td>
+    </tr>
+  </tbody>
+</table>
+  
+    <br><hr><br>
+  
+    <table class="table table-dark">
+  <thead>
+  	<tr>
+    <th colspan="4"><center>Last 10 Payments</center></th>
+    </tr>
+    <tr>
+      <th scope="col">Time & Date</th>
+      <th scope="col" colspan="2">Transaction ID</th>
+     <th scope="col">amount</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><?php print $payments->rows[0]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[0]->txId; ?></td>
+      <td><?php print $payments->rows[0]->amount; print " LTC"; ?></td>
+    </tr>
+        <tr>
+      <td><?php print $payments->rows[1]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[1]->txId; ?></td>
+      <td><?php print $payments->rows[1]->amount; print " LTC"; ?></td>
+    </tr>
+      <tr>
+      <td><?php print $payments->rows[2]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[2]->txId; ?></td>
+      <td><?php print $payments->rows[2]->amount; print " LTC"; ?></td>
+    </tr>
+      <tr>
+      <td><?php print $payments->rows[3]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[3]->txId; ?></td>
+      <td><?php print $payments->rows[3]->amount; print " LTC"; ?></td>
+    </tr>
+      <tr>
+      <td><?php print $payments->rows[4]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[4]->txId; ?></td>
+      <td><?php print $payments->rows[4]->amount; print " LTC"; ?></td>
+    </tr>
+      <tr>
+      <td><?php print $payments->rows[5]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[5]->txId; ?></td>
+      <td><?php print $payments->rows[5]->amount; print " LTC"; ?></td>
+    </tr>
+      <tr>
+      <td><?php print $payments->rows[6]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[6]->txId; ?></td>
+      <td><?php print $payments->rows[6]->amount; print " LTC"; ?></td>
+    </tr>
+      <tr>
+      <td><?php print $payments->rows[7]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[7]->txId; ?></td>
+      <td><?php print $payments->rows[7]->amount; print " LTC"; ?></td>
+    </tr>
+      <tr>
+      <td><?php print $payments->rows[8]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[8]->txId; ?></td>
+      <td><?php print $payments->rows[8]->amount; print " LTC"; ?></td>
+    </tr>
+      <tr>
+      <td><?php print $payments->rows[9]->timestamp; ?></td>
+      <td colspan="2"><?php print $payments->rows[9]->txId; ?></td>
+      <td><?php print $payments->rows[9]->amount; print " LTC"; ?></td>
+    </tr>
+  </tbody>
+</table>
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
```

Comment:
please also use local libaries - feel free to add a new global css and js folder